src/components/CountriesList.js

```diff
@@ -0,0 +1,13 @@
+import { Link } from 'react-router-dom';
+
+export default function CountriesList(props) {
+    return (
+        <div className="col-5" style={{maxHeight: '90vh', overflow: 'scroll'}}>
+            <div className="list-group">
+                {props.countries.map(country => 
+                    <Link to={`/${country.cca3}`} className="list-group-item list-group-item-action">{country.flag} {country.name.common}</Link>
```

Comment:
You can add a key for the `Link` tag `key={country.cca3}` to avoid the warning in the developer's tools console.

`Warning: Each child in a list should have a unique "key" prop.`